src/app/dashboard/dashboard.component.ts

```diff
@@ -1,8 +1,9 @@
-import { Location } from '@angular/common';
-import { Component, OnInit } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
-import { LastBuild } from '../models/last-build';
-import { ProjectsBuild } from '../models/projects-build';
+import {Location} from '@angular/common';
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {LastBuild} from '../models/last-build';
+import {ProjectsBuild} from '../models/projects-build';
+import {JobStatus} from '../models/job';
```

Comment:
nit: do the first 5 lines change in any significant way?

Reply:
No, my ide seems to have collapsed the whitespaces :/